assets/charts/components/azure-arc-lokomotive/templates/rbac.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: azure-arc-onboarding
+subjects:
+- kind: ServiceAccount
+  name: azure-arc-onboarding
+  namespace: {{ .Values.namespace }}
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: cluster-admin
```

Comment:
We could limit that :)

Reply:
https://docs.microsoft.com/en-us/azure/architecture/hybrid/arc-hybrid-kubernetes#cluster-registration

excerpt
```
You'll need a kubeconfig file to access the cluster and cluster-admin role on the cluster
for deploying Arc-enabled Kubernetes agents. 
```

Reply:
Can we put this link in the manifest?

Reply:
done.